deepchem/models/torch_models/pinns_model.py

```diff
@@ -0,0 +1,191 @@
+import torch
+import torch.nn as nn
+from typing import Callable, Dict, List, Optional, Union
+from deepchem.models.torch_models.torch_model import TorchModel
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class NeuralNet(nn.Module):
+    """A simple neural network that will be used by the PINNModel when no default model is provided.
+    """
+
+    def __init__(self, in_channels: int = 1):
+        super(NeuralNet, self).__init__()
+        self.net = nn.Sequential(nn.Linear(in_channels, 64), nn.Tanh(),
+                                 nn.Linear(64, 64), nn.Tanh(), nn.Linear(64, 1))
+
+    def forward(self, x):
+        if not isinstance(x, torch.Tensor):
+            x = torch.tensor(x, dtype=torch.float32)
+        return self.net(x)
+
+
+class PINNModel(TorchModel):
```

Comment:
Pls add some citations and details about this model.

Reply:
Done.